yb-voyager/src/srcdb/postgres.go

```diff
@@ -71,24 +71,61 @@ var PG_COMMAND_VERSION = map[string]string{
 	"psql":       "9.0", //psql features we need are available in 7.1 onwards, keeping it to 9.0 for safety
 }
 
-const FETCH_COLUMN_SEQUENCES_QUERY_TEMPLATE = `SELECT
+/*
+This query returns all types of sequences attached to the tables -
+1. SERIAL/BIGSERIAL datatypes
+2. IDENTITY (ALWAYS / DEFAULT) columns
+3. SEQUENCE objects owned by the table columns
+
+this query doesn't return sequences that are attached to table by using the nextval function of sequence as DEFAULT clause of column
+e.g. CREATE TABLE default_clause_tbl(id int DEFAULT nextval('public.seq'::regclass), val text);
+*/
+const FETCH_COLUMN_ALL_SEQUENCES_EXCEPT_DEFAULT_QUERY_TEMPLATE = `SELECT
+(tn.nspname || '.' || t.relname)  AS table_name,
 a.attname AS column_name,
-COALESCE(seq.relname, '') AS sequence_name,
-COALESCE(ns.nspname, '') AS schema_name
-FROM pg_class AS t
-JOIN pg_attribute AS a ON a.attrelid = t.oid
-JOIN pg_namespace AS tn ON tn.oid = t.relnamespace
-LEFT JOIN pg_attrdef AS ad ON ad.adrelid = t.oid AND ad.adnum = a.attnum
-LEFT JOIN pg_depend AS d ON d.objid = ad.oid
-LEFT JOIN pg_class AS seq ON seq.oid = d.refobjid
-LEFT JOIN pg_namespace AS ns ON ns.oid = seq.relnamespace
+seq_ns.nspname AS sequence_schema,
+seq.relname AS sequence_name
+FROM pg_class t
+JOIN pg_namespace tn ON tn.oid = t.relnamespace
+JOIN pg_attribute a ON a.attrelid = t.oid
+LEFT JOIN pg_attrdef ad ON ad.adrelid = t.oid AND ad.adnum = a.attnum
+LEFT JOIN pg_depend dep
+	ON (dep.objid = ad.oid ) 
+	OR (a.attidentity <> '' AND dep.refobjid = a.attrelid AND dep.refobjsubid = a.attnum)  -- identity sequence
+	OR (dep.refobjid = t.oid AND dep.refobjsubid = a.attnum) -- owned sequences
+LEFT JOIN pg_class seq ON seq.oid = dep.objid AND seq.relkind = 'S'
```

Comment:
Most of the part of the queries are same.
please point out  the differences in the queries with a `-- comment`

Reply:
added a comment before the query what is the difference